src/poliastro/plotting.py

```diff
@@ -101,14 +102,19 @@ def set_attractor(self, orbit):
 
         self.ax.add_patch(mpl.patches.Circle((0, 0), radius, lw=0, color=color))
 
-    def plot(self, orbit, osculating=True, label=None):
+    def plot(self, orbit, osculating=True, label=None, propagate=False):
         """Plots state and osculating orbit in their plane.
 
         """
         # TODO: This function needs a refactoring
         if not self._frame:
             self.set_frame(*orbit.pqw())
 
+        # Propagates backwards of forwards to the last added orbit epoch.
+        if propagate:
+            if self._orbits:
+                orbit = orbit.propagate(self._orbits[-1].epoch - orbit.epoch)
```

Comment:
What happens if we propagate some orbits, and leave as is some others? And also, are we allowing the user to propagate to any of the epochs that are in the `OrbitPlotter`?

I am not convinced that this is the right approach. Perhaps we can it tomorrow during the meeting?